libs/auth/src/java_card_config.ts

```diff
@@ -15,36 +18,62 @@ export interface JavaCardConfig {
   vxCertAuthorityCertPath: string;
 }
 
+/**
+ * Should be kept in sync with vxsuite-complete-system scripts
+ */
+const PROD_CONFIG_DIRECTORY = '/vx/config';
```

Comment:
I'd love for this to be an environment variable for configuration. Not required in this PR, but just food for thought.

Reply:
Ah nice, I like that - will make it such that we don't have to keep vxsuite in sync with vxsuite-complete-system. It'll just be vxsuite-complete-system's responsibility to be internally consistent

Reply:
https://github.com/votingworks/vxsuite/pull/3226/commits/55aaca6ebb915c34f5944cb9fd8c28b0a6e48d6a